### ISSUE TITLE
Code Freeze to fix Major Issues

### ISSUE THREAD

**Issue by TedStudley** [TedStudley/mc-mini#21]
@hlokavarapu @icherkashin @egpuckett 

I want to tackle the code formatting and output folder removal issues, which are both highly likely to create major conflicts with anybody's local changes.

In order to make sure that we don't mess up a bunch of everybody's work in the process, I'm going to call for a code freeze. We'll set a deadline for any in-progress code to be merged into master, and after that nothing will be merged in until the major changes have been completed. After the major changes, any code which wasn't merged in before the code freeze (hopefully nothing!) will have to be fixed in order to remove all merge conflicts. This will likely go better for everybody involved than having to suddenly deal with a bunch of unexpected merge conflicts.

To plan this, I need to know what everybody is working on, in what parts of the code, and how long it will take to get into a state which can be merged in. We can use the code freeze milestone to track all of the things that need to be finished before moving forward with this.

**Comment by icherkashin** [comment]
I'm afraid no one else is working on anything at the moment, but you should ask Prof. Puckett @egpuckett if he's still working with Jonathan and Harsha.

I'll be working on the indices and velocity boundary dimensions issue. No new functionality will be added, so this should not cause any conflicts. I'm planning to finish this during the Thanksgiving break.

**Comment by TedStudley** [comment]
It appears as though the following command should remove all traces of the `output` folder from all commits in the codebase:
```
git filter-branch --index-filter 'git rm --cached -r --ignore-unmatch output' --prune-empty --tag-name-filter cat -- --all
```
this will *definitely* cause merge conflicts, which should hopefully be fairly simply fixed. This should reduce the size of the repository, but I have the feeling that we may have to jump through a couple extra hoops in order to actually reduce the size.

**Comment by icherkashin** [comment]
>this will definitely cause merge conflicts, which should hopefully be fairly simply fixed. This should reduce the size of the repository, but I have the feeling that we may have to jump through a couple extra hoops in order to actually reduce the size.

I wouldn't object to this.

**Comment by TedStudley** [comment]
I think I've already made the update... The original repository download doesn't appear to be significantly smaller, although it's possible that I've missed something. It's likely that I'll have to make the same changes to (or delete) any other branches of the codebase, as well.

**Comment by TedStudley** [comment]
Closing this issue since the code freeze no longer seems to be necessary.

**Comment by icherkashin** [comment]
The issue with indexing and velocity boundary values turned out to be more complex that I initially thought, so that's postponed also.
